oximeter/oximeter/src/types.rs

```diff
@@ -1032,4 +1152,11 @@ mod tests {
             Err(MetricsError::DuplicateFieldName { .. })
         ));
     }
+
+    #[test]
+    fn test_foo() {
```

Comment:
This looks maybe like it was a leftover test from locally doing development? 

I also name my tests `test_asdf` before I know what to do with them...

Reply:
Ha, thanks for catching that! I could have sworn I removed it, I'll do that now.